Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -368,6 +376,112 @@ extension BrowserViewController: BrowserToolbarDelegate {
     }
 }
 
+private var bars = [SnackBar]()
```

Comment:
Hmm.. The BVC one is a list of visible bars, but we could use the Browser's list for that I think. It still makes it a bit hard. "Get all the bars after this one" requires getting the list, getting the index of this bar, and then iterating over that sublist... I made this change. Will try to clean it up a bit and send it that way.

This actually should get much easier when the Browser is a UIViewController. It can just add these to itself.